app/routes/index.tsx

```diff
@@ -24,11 +24,13 @@ import CallToAction from "~/components/CallToAction";
 import RootLayout from "~/components/RootLayout";
 import Wave from "~/images/Wave";
 
+import gradientStyles from "~/styles/gradient.css";
 import headerStyles from "~/styles/header.css";
 import projectsStyles from "~/styles/projects-list.css";
 
 export function links() {
   return [
+    { rel: "stylesheet", href: gradientStyles },
```

Comment:
**issue:** the `gradient.css` file will only be imported on the index page, but what we really need is for it to be imported everywhere, so this line should probably move to the `root.tsx` file. For example, the footer on [the about page](https://opensourcehub-git-eng-527-update-footer-codesee.vercel.app/about) doesn't show the gradient.

This is because, in Remix, [stylesheets are only imported by the route boundaries that require them](https://remix.run/docs/en/v1/guides/styling). So for example, the CSS in `header.css` and `projects-list.css` will only show up on the home page.

Reply:
Thanks. It was updated and footer style is up on every page.